openlibrary/core/search.py

```diff
@@ -0,0 +1,179 @@
+from __future__ import absolute_import
+
+import urllib
+import urllib2
+import web
+import simplejson
+import six
+
+from infogami.utils import delegate
+
+from openlibrary.utils import dateutil
+from openlibrary.core.models import Edition
+from openlibrary.core.helpers import get_coverstore_url
+from openlibrary.core.helpers import bookreader_host
+from openlibrary.core.lending import config_http_request_timeout
+
+
+PRESET_QUERIES = {
+    'preset:thrillers': '(creator:"Clancy, Tom" OR creator:"King, Stephen" OR creator:"Clive Cussler" OR creator:("Cussler, Clive") OR creator:("Dean Koontz") OR creator:("Koontz, Dean") OR creator:("Higgins, Jack")) AND !publisher:"Pleasantville, N.Y. : Reader\'s Digest Association" AND languageSorter:"English"',
+    'preset:children': '(creator:("parish, Peggy") OR creator:("avi") OR title:("goosebumps") OR creator:("Dahl, Roald") OR creator:("ahlberg, allan") OR creator:("Seuss, Dr") OR creator:("Carle, Eric") OR creator:("Pilkey, Dav"))',
+    'preset:comics': '(subject:"comics" OR creator:("Gary Larson") OR creator:("Larson, Gary") OR creator:("Charles M Schulz") OR creator:("Schulz, Charles M") OR creator:("Jim Davis") OR creator:("Davis, Jim") OR creator:("Bill Watterson") OR creator:("Watterson, Bill") OR creator:("Lee, Stan"))',
```

Comment:
These presets queries probably need some work at some point. It feels like they should be config rather than code. I thought they _were_ config previously, but I see you moved them from `home.py`. Having them here because they are IA related makes sense, but the naming `search.py` makes it hard to locate or figure out what this file is for. The previous location was at least clear that it related to to home page. I agree with @cdrini 's comment that this needs a rename to make it clear that it related to the IA elastic search interface.   Core search implies it is OL core search ;)

Reply:
not going to fix in this pr